torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -4936,6 +4936,54 @@ def _test_post_localSGD_optimizer_parity(self, create_averager, grad_is_view):
         def _create_periodic_model_averager(self):
             return averagers.PeriodicModelAverager(period=4, warmup_steps=10)
 
+        def _test_post_localSGD_optimizer_step_reload(self, create_averager):
+            learning_rate = 0.03
```

Comment:
Is it possible to refactor the code common with `_test_post_localSGD_optimizer_parity()` into a helper method?

There is significant overlap.

Reply:
+1, or we can file a follow up issue

Reply:
Refactored common code